client/web/src/hooks/self-update.ts

```diff
@@ -58,7 +58,7 @@ export function useInstallUpdate(currentVersion: string, cv?: VersionInfo) {
     let tsAwayForPolls = 0
     let updateMessagesRead = 0
 
-    let timer = 0
+    let timer: NodeJS.Timeout
```

Comment:
Changes in this file fix failing lint test https://github.com/tailscale/tailscale/actions/runs/7118168197/job/19380476663?pr=10484